packages/advanced-logic/specs/payment-network-erc20-fee-proxy-contract-0.1.0.md

```diff
@@ -265,6 +258,40 @@ the 'addFee' event:
 | **parameters.feeAddress** | `feeAddress` from parameters    |
 | **parameters.feeAmount**  | `feeAmount` from parameters     |
 
+## Action: declarePayment
```

Comment:
Imho, It's weird to have only declarePayment and not declareRefund. 
We should add it in the same time.